### DISCUSSION TITLE
Welcome to Questions-you-want-answers-to Discussions!

### DISCUSSION THREAD

**Discussion by Sephrim-NightShade** [Sephrim-NightShade/Questions-you-want-answers-to#2] [General]
Here is were we talk to each other. Yes talking is needed.

**Comment by Cosmikality** [comment]
no

**Comment by Sephrim-NightShade** [comment]
こにちわ

**Comment by Cosmikality** [comment]
como muy deliciosos

**Reply by Sephrim-NightShade** [reply]
Me no comprehende Espanol, amigo.

**Comment by Sephrim-NightShade** [comment]
おひよ

**Reply by Sephrim-NightShade** [reply]
lol

**Comment by Alexander3531** [comment]
Gay

**Reply by Sephrim-NightShade** [reply]
But i'm bisexual

**Reply by Sephrim-NightShade** [reply]
Bum, Bum, bum.

**Comment by Cosmikality** [comment]
tu estupido

**Reply by Sephrim-NightShade** [reply]
Yes

**Comment by Sephrim-NightShade** [comment]
Stipido Regazzo Americano

**Reply by Sephrim-NightShade** [reply]
Yes.

**Comment by Sephrim-NightShade** [comment]
Anytime ya'll need something, just let me know.

**Reply by Sephrim-NightShade** [reply]
I can't help ya if you don't ask me.

**Comment by Sephrim-NightShade** [comment]
Are ya'll fine with me inviting Billy? He is very smart, but has trouble focusing on the tasks at hand. I need your input on this guys, since he'll be working with you all more. Just let me know.

**Comment by Sephrim-NightShade** [comment]
Any of you there?

**Comment by Sephrim-NightShade** [comment]
Figures.

**Comment by Sephrim-NightShade** [comment]
真剣にみんな、お互いに話し合ってください。
Shinken ni min'na, otagai ni hanashiatte kudasai.

**Comment by Sephrim-NightShade** [comment]
I would like to inform each of you that I did not create this whole profile and organization just to make a discord-style chat place for us. We are supposed to be working, and it seems by each of your guy's activity log on your accounts that none of you have added or even begun to work. (Except for Cosmikality, it seems he may be at work on the other branches but I cannot confirm this with sufficient accuracy.) Please, guys, all I ask is that you screw around with the coding and all and at least make some progress. If not, then a polite message pointing me to someone who will work will be appreciated. I hired you guys because I literally cannot do this on my own. Hope you guys even read this. 
Your friend, Sephrim, over and out.

**Comment by Sephrim-NightShade** [comment]
GUYS! Are any of you even taking me seriously?
